public/service-worker.js

```diff
@@ -0,0 +1,44 @@
+const cacheName = "v1";
+
+// Install a service worker
+self.addEventListener("install", (event) => {
+    console.log("Service Workers: Installed");
+});
+
+// Cache and return requests
+self.addEventListener("fetch", (event) => {
+    event.respondWith(
+        fetch(event.request)
+            .then((res) => {
+                //Make clone of response
+                const resClone = res.clone();
+                // Open cache
+                caches.open(cacheName).then((cache) => {
+                    // Add response to the cache
+                    cache.put(event.request, resClone);
```

Comment:
Now we're caching the requests on the service worker but what about caching the assets? We need to find a way to start caching the assets with this create-react-app. wdyt @calovey ?